classes/02class/exercises/c02-network06/mfreitassm/README.md

```diff
@@ -0,0 +1,102 @@
+# c02-network06
+
+## Commands Execution Output
+
+- Commands for creating the NAT gateway:
+```
+Create a network interface
+//subnet public-b
+aws ec2 create-network-interface --subnet-id subnet-0dd087be76dd34768 --description "devopsacademy network interface" --groups sg-021fa8202867b5ce0 
```

Comment:
Hi, no need to create a network interface for this exercise.

Reply:
Okay